src/app/personside/infotabs/meldinger/utils/meldingerUtils.ts

```diff
@@ -96,7 +96,9 @@ export function erMeldingSpørsmål(meldingstype: Meldingstype) {
 export function erKontorsperret(traad: Traad): boolean {
     return !!eldsteMelding(traad).kontorsperretEnhet;
 }
-
+export function kanJournalfores(meldingstype: Meldingstype): boolean {
```

Comment:
Her må du se på koden i modiabrukerdialog for å være sikker på at du har fått med deg alle kriteriene (\modiabrukerdialog\sporsmalogsvar\src\main\java\no\nav\sbl\dialogarena\sporsmalogsvar\lamell\haandtermelding\HaandterMeldingValgPanel.java)